### PR TITLE
Movement Direction Modeling

### PR DESCRIPTION
**Pull request by dragonjws**

### CODE REVIEW

File: Model.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ class Model {
     // "Source of Truth"
     snake = [];   // example: [[1,1], [2,1], [2,2]]
 
+    xDirection = 1;
+
```
Comment (jadowdl) [inline Model.js]: probably don't need a space here.

Comment (westonz7042) [inline Model.js]: spicy

File: Model.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ class Model {
...
+    yDirection = 0;
```
Comment (itblursby) [inline Model.js]: What does that mean?  Does xDirection = 1 mean left and 0 mean right?

File: Model.js [hunk 3]
```diff
@@ -9,6 +9,9 @@ class Model {
     // "Source of Truth"
     snake = [];   // example: [[1,1], [2,1], [2,2]]
 
+    xDirection = 1;
```
Comment (Imperii) [inline Model.js]: LGTM

**Pull request merged**